flink/v1.14/flink/src/test/java/org/apache/iceberg/flink/sink/TestFlinkIcebergSinkV2.java

```diff
@@ -155,10 +159,6 @@ private void testChangeLogs(List<String> equalityFieldColumns,
                               List<List<Record>> expectedRecordsPerCheckpoint) throws Exception {
     DataStream<Row> dataStream = env.addSource(new BoundedTestSource<>(elementsPerCheckpoint), ROW_TYPE_INFO);
 
-    // Shuffle by the equality key, so that different operations of the same key could be wrote in order when
-    // executing tasks in parallel.
-    dataStream = dataStream.keyBy(keySelector);
```

Comment:
Yes, because we ensure the input will not be disorder in `FlinkSink.forMapperOutputType` and data will be correctly  distributed by equaily fields or partition fields in different distribution mode. So we can remove this in unittest